src/hooks/useSendEmailNotification.ts

```diff
@@ -0,0 +1,85 @@
+import { fetch } from '@inrupt/solid-client-authn-browser'
+import { useMutation } from '@tanstack/react-query'
+import { communityId, emailNotificationsService } from 'config'
+import { useCallback } from 'react'
+import { useProfile } from './data/useProfile'
+
+type Person = { id: string; name: string }
+type MessageNotification = { messageId: string; message: string }
+type MessageNotificationData = {
+  from: Person
+  to: Person
+  messageId: string
+  message: string
+}
+
+const getMessageNotificationBody = ({
+  from,
+  to,
+  messageId,
+  message,
+}: MessageNotificationData) => ({
+  '@context': 'https://www.w3.org/ns/activitystreams',
+  type: 'Create',
+  actor: {
+    type: 'Person',
+    id: from.id,
+    name: from.name,
+  },
+  object: {
```

Comment:
I know it doesn't matter much, but I find the order of the objects here a bit confusing. I'd use:

```
actor: {...},  // from
target: {...},  // to
object: {...}  // the message
```
instead, as in the parameters list.